src/Arcus.WebApi.Security/Authorization/Jwt/JwtTokenReader.cs

```diff
@@ -17,10 +20,24 @@ public class JwtTokenReader : IJwtTokenReader
             "https://login.microsoftonline.com/common/v2.0/.well-known/openid-configuration";
 
         private readonly string _applicationId;
+        private readonly ClaimCheck _claimCheck;
         private readonly TokenValidationParameters _tokenValidationParameters;
         private readonly JwtSecurityTokenHandler _handler = new JwtSecurityTokenHandler();
         private readonly ConfigurationManager<OpenIdConnectConfiguration> _configManager;
 
+        /// <summary>
+        ///     Constructor
+        /// </summary>
+        /// <remarks>Uses Microsoft OpenId connect discovery endpoint</remarks>
+        /// <param name="claimCheck">Custom claims key-value pair to validate against</param>
+        public JwtTokenReader(ClaimCheck claimCheck)
```

Comment:
Do we need a custom type for this or can we just pass a dictionary? This feels a bit less overkill, no?

Reply:
I thought about this option, however, I followed the proposal made in the bug, where the creation of a ClaimCheck object was mentioned. From my point of view, it is not necessary, I just followed that indication.